contrib/kind.sh

```diff
@@ -543,8 +543,8 @@ set_default_params() {
   SVC_CIDR_IPV6=${SVC_CIDR_IPV6:-fd00:10:96::/112}
   JOIN_SUBNET_IPV4=${JOIN_SUBNET_IPV4:-100.64.0.0/16}
   JOIN_SUBNET_IPV6=${JOIN_SUBNET_IPV6:-fd98::/64}
-  MASQUERADE_SUBNET_IPV4=${MASQUERADE_SUBNET_IPV4:-169.254.169.0/29}
```

Comment:
NIT: In the future, please extract renames into a separate commit. It makes reviewing the other changes easier.

Reply:
I will keep this suggestion in mind , Thanks :)